src/renderer/preload-main.ts

```diff
@@ -76,6 +76,7 @@ if (ssfWindow.ssf) {
         registerRestartFloater: ssfWindow.ssf.registerRestartFloater,
         setCloudConfig: ssfWindow.ssf.setCloudConfig,
         checkMediaPermission: ssfWindow.ssf.checkMediaPermission,
+        capabilities: {bug19017: true}, // flag to enable RTC patch fixing electron bug https://github.com/electron/electron/issues/19017
```

Comment:
do we need this fix for 1.5 as well? If so, we'll need to add this in another place and use that instead -> https://github.com/symphonyoss/SymphonyElectron/blob/5726a7656bf77b46f94f0c74b058f22a2d2bd3d4/src/renderer/ssf-api.ts#L511

Reply:
We do, but this seems to work on 1.5 when I test locally. the window.manaSSF object is exposed on both.

Reply:
Yeah, it does because `manaSSF` is an additional object anyway. But, for 1.5, we streamline everything through `window.ssf`. So, better to use that to be aligned consistently with the other APIs.

Reply:
okay. so should I move this to ssf-api as a public property instead?

Reply:
Yes please.